File: rb/test/exists_test.rb
# PollinationsAi SDK exists test

require "minitest/autorun"
require_relative "../PollinationsAi_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = PollinationsAiSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end

File: ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { PollinationsAiSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await PollinationsAiSDK.test()
    equal(null !== testsdk, true)
  })

})
